models/tt_transformers/tt/model.py

```diff
@@ -361,6 +369,7 @@ def ttnn_decode_forward(
         page_table=None,
         kv_cache=None,
         argmax_on_device=False,
+        **kwargs,
```

Comment:
This makes it hard to see which arguments could be expected. Why not list the potential arguments with default values above?